analytics/views.py

```diff
@@ -1126,6 +1126,9 @@ def support(request: HttpRequest) -> HttpResponse:
             do_change_plan_type(realm, new_plan_type)
```

Comment:
`do_change_plan_type` already creates `RealmAuditLog` objects. What we should be doing here would be to make `do_change_plan_type` take `acting_user` as an optional argument and use that in `RealmAuditLog` creation.

See `do_change_logo_source` for an example. So we don't have to create the RealmAuditLog objects here.